src/cli/javascript/runtime.rs

```diff
@@ -45,8 +45,9 @@ impl WorkerIO<Message, Message> for Runtime {
         log::debug!("event: {:?}", event);
         let command = LOCAL_RUNTIME.with(move |cell| {
             let script = self.script.clone();
+            // TODO: use `get_or_try_init`. Currently it is an unstable feature
             cell.borrow()
-                .get_or_init(|| LocalRuntime::try_new(script).unwrap());
+                .get_or_init(|| LocalRuntime::try_new(script).expect("JS runtime not initialized"));
```

Comment:
This could basically mean that the script is invalid. Let's add some tests for it also.